employee_project/employee.py

```diff
@@ -86,3 +86,24 @@ def get_emp_salary_input(msg = "Enter employee salary:"):
         print("Invalid input, enter a valid input ex: 20000")
         sal = get_emp_salary_input(msg)
     return sal
+
+
+def employee_name(employee_name):
+    """ @param: string
+        @desc: method returns the valide name
+        @returns: string
+    """
+    while True:
+        Name_min = 3
+        Name_max = 15
+        employee_name = input("Enter the  employee_name:")
```

Comment:
Don't give the description employee_name like this 

"Enter the  Employee Name:"

Reply:
removed from the main.py